pkg/subctl/cmd/root.go

```diff
@@ -60,7 +60,7 @@ func init() {
 
 func addKubeconfigFlag(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "absolute path(s) to the kubeconfig file(s)")
-	cmd.PersistentFlags().StringVar(&kubeContext, "kubecontext", "", "kubeconfig context to use")
+	cmd.PersistentFlags().StringVar(&kubeContext, "kubecontext", "", "comma separated list of kubeconfig contexts to use")
```

Comment:
Can we explain the default behavior? I assume "If not specified, all contexts referenced by kubeconfig are used"